src/pyhf/utils.py

```diff
@@ -41,6 +42,24 @@ def convert(self, value, param, ctx):
             self.fail(f'{value:s} is not a valid equal-delimited string', param, ctx)
 
 
+class VolumeMountPath(click.Path):
```

Comment:
As this is a public class should we add a docstring and at least add it to `__all__`?

Reply:
can't be used outside of click (and not sure why we want to document it right now for users). I hope nobody needs this outside of our code.

Reply:
That's a good point (RE: needing to be used witch Click). Should we just make this "Python private" then and name it `_VolumeMountPath`?

Reply:
We didn't do the same with `EqDelimStringParamType`.

Reply:
Yeah. We can revise this later if needed.